lib/spack/llnl/util/tty/__init__.py

```diff
@@ -18,6 +19,10 @@
 _debug = False
 _verbose = False
 _stacktrace = False
+_suppress_msg = False
+_suppress_info = False
+_suppress_warn = False
+_suppress_error = False
```

Comment:
Can you make the sense of these positive?  i.e., `if _print_msg:` instead of `if not _suppress_msg:`